subsys/bluetooth/services/ddfs.c

```diff
@@ -177,6 +178,7 @@ static ssize_t dm_write_ctrl_point(struct bt_conn *conn, const struct bt_gatt_at
 {
 	const struct write_ctrl_point_req *req = buf;
 	struct bt_ddfs_dm_config dm_config;
+	uint8_t config_arr[2];
```

Comment:
I see that you are using net_buf in this module. Consider using it to encode this simple array. The ```net_buf_simple_add``` API removes the use of magic numbers for array indices and seems more elegant

Reply:
Provide a defintion for the configuration array size (or net_buf size)

Reply:
ACK